components/server/src/workspace/workspace-starter.ts

```diff
@@ -1678,6 +1684,12 @@ export class WorkspaceStarter {
                     subjectID: "*",
                     operations: ["create", "get"],
                 }),
+            "resource:" +
```

Comment:
Do we really need to provide access and expose info about org settings to workspaces? So far we were not doing it. Alternatively we can inject `GITPOD_WORKSPACE_DEFAULT_IMAGE` as an env var.

Would be expect changing behavoiur after the worskpace startup?

cc @geropl @svenefftinge

Reply:
If it's a env var, then it can't react to real org settings immediately. But this case can be rarely happen, use env var can make whole process easier ()

Reply:
I just not sure that we should give a workspace more permissions just because we want to propagate the default workspace image. We could start with explicit env var for the image and reconsider later?

Reply:
hm, from another side especially at the beginning while you are on boarding and using `gp validate` it would be nice to iterate without restarting a workspace each time. Also this permission does not seem to give more besides reading org settings, does it allow something else?

cc @geropl @svenefftinge wdyt?

Reply:
@akosyakov  Addressed https://github.com/gitpod-io/gitpod/pull/18723/commits/f875e63b0783572ecc7fb5a85499f1e45ef27562